### PR TITLE
CC Waterfall plot with META Income Statement

### PR DESCRIPTION
**Pull request by soheilkashan**
Hi team - hope all is good. the code runs locally but can't render the book locally. Please let me know if I missed something.

### CODE REVIEW

**Comment by zcysxy** [comment]
What's the error when you render the book locally? It is highly likely that if it does not render locally, it will not render online either

**Comment by soheilkashan** [comment]
maybe this is not relevant but when try to knit I get this error:
```
Error in gzfile(file, "rb") : cannot open the connection
Calls: <Anonymous> ... render_new_session -> clean_meta -> readRDS -> gzfile
In addition: Warning message:
In gzfile(file, "rb") :
  cannot open compressed file 'cc22.rds', probable reason 'No such file or directory'
Execution halted```

File: waterfall_plot_meta_income_statement.Rmd [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+#### Data Source
+
+In this short tutorial we will be exploring **META**'s quarterly income statement and use Waterfall charts to visualize their Q3 - 2022 financial statement.
+
+META Quarterly Earnings reports: <https://investor.fb.com/financials/default.aspx>
+  
+  #### Data prep
+  
+  ```{r}
```
Comment (jtr13) [inline waterfall_plot_meta_income_statement.Rmd:34]: remove all spaces before ```

**Comment by jtr13** [comment]
The problem I believe is that there are some code chunks with an extra spaces before ` ```{r} ` which is preventing the document from knitting although the code works in the interactive session. Please remove those extra spaces and we'll see if it works.

File: waterfall_plot_meta_income_statement.Rmd (hunk 1 above)
Comment (soheilkashan) [reply waterfall_plot_meta_income_statement.Rmd]: done

File: waterfall_plot_meta_income_statement.Rmd [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+From the waterfall chart we can observe that Q3-2022 income is significantly lower 
+than the last year. This is a mainly driven by the increase in expenses in research and 
+development, and the decrease in revenue from advertising. 
+
+
+##### Using plotly to create simple waterfall
+Alternatively we can use plotly to create interactive waterfall charts.
+https://plotly.com/r/waterfall-charts/ 
+  
+  ``` {r}
```
[the remark this answers is not in the document]
Comment (soheilkashan) [reply waterfall_plot_meta_income_statement.Rmd]: done

**Comment by soheilkashan** [comment]
@jtr13 thank you for the comments. I have removed the errand spaces.

**Comment by jtr13** [comment]
Thanks for your contribution! 🎉

**Pull request merged**